.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions: # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+
+    steps:
+      # Seed the build number with last number from TeamCity.
+      # This env var is used by the JS, and SBT builds, and guardian/actions-riff-raff.
+      # Set the value early, rather than `buildNumberOffset` in guardian/actions-riff-raff, to ensure each usage has the same number.
+      # For some reason, it's not possible to mutate GITHUB_RUN_NUMBER, so set BUILD_NUMBER instead.
+      - name: Set BUILD_NUMBER environment variable
+        run: |
+          LAST_TEAMCITY_BUILD=145
+          echo "BUILD_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "adopt"
+          java-version: "8"
+          cache: "sbt"
+
+      - name: AWS Auth
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+
+      - name: Build and Test project
+        run: |
+          sbt clean compile test universal:packageBin
+          cp target/universal/snapshotter-lambda.zip ./snapshotter-lambda.zip
+
+      - uses: guardian/actions-riff-raff@v2
```

Comment:
As you're using `actions-riff-raff` instead of deprecated `sbt-riffraff-artifact` you can remove this plugin from [`built.sbt`](https://github.com/guardian/flexible-snapshotter/blob/main/build.sbt#L37-L45) and [`plugins.sbt`](https://github.com/guardian/flexible-snapshotter/blob/main/project/plugins.sbt#L3).